__tests__/app.test.js

```diff
@@ -109,4 +109,30 @@ describe("/api/articles/:article_id/comments", () => {
             expect(body.msg).toEqual("article not found!")
         })
     })
+    const comment = {
+        "username": "lurker",
+        "body": "added comment",
+    }
+    test(":) POST 201 - posts comment given article id, returns posted comment", () => {
```

Comment:
Have a think about another 201 test - check that you ignore unnecessary properties - e.g. post a body with username, body, and an additional key that isn't required.